dts/bindings/sensor/rohm,bh1750-i2c.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 Lucas Heitzmann Gabrielli
```

Comment:
rename file to just `rohm,bh1750.yaml`.  We use the `-i2c` when a device supports different buses like `i2c` and  `spi`.